project.clj

```diff
@@ -50,7 +50,7 @@
                              :linux   "xdg-open"}}}
 
   :aliases {"dev"          ["with-profile" "dev" "do"
-                            ["run" "-m" "shadow.cljs.devtools.cli" "watch" "main" "renderer"]]
+                            ["run" "-m" "shadow.cljs.devtools.cli" "watch" "main" "renderer" "find-in-page"]]
             "compile"        ["with-profile" "dev" "do"
```

Comment:
@arkRedM I see why my build was failing. The reason is I use `yarn dev` more often than `lein dev`. Having two shadow-cljs wrappers from yarn and lein is confusing and may lead to errors. They do subtlely different things, though, I think. May be good to address this issue in a later PR.

If this is only in the `dev` build, we need to make sure the new `:find-in-page` is also released with the production build.

Reply:
Need to look into what exactly is diff between yarn and lein dev builds. But yeah, should add it to prod.

Reply:
Looks like it got fixed in the recent merge https://github.com/athensresearch/athens/pull/1246.  
mark it resolved?